apps/jobs/views.py

```diff
@@ -289,11 +297,63 @@ class UserViewSets(viewsets.ModelViewSet):
     queryset = User.objects.all()
     serializer_class = UserSerializer
 
-    def create(self, request, *args, **kwargs):
+    def update(self, request, *args, **kwargs):
         """
         Overriding the create method (used in POST request),
         This method creates a new user profile in the database.
         """
+        common_response_parameters = {
```

Comment:
put in constant package

Reply:
Moved this dictionary and other necessary response related items in `constants/response.py`